src/test/java/io/vertx/core/http/Http2ServerTest.java

```diff
@@ -2238,6 +2239,60 @@ public void onHeadersRead(ChannelHandlerContext ctx, int streamId, Http2Headers
     await();
   }
 
+  @Test
+  public void test103EarlyHints() throws Exception {
```

Comment:
this test should be moved to the HttpTest class so it tests both HTTP/1.1 and HTTP/2

Reply:
it sounds reasonable to use vertx HTTP server and client in the same test

Reply:
ping @cavallium

Reply:
Sorry, I had forgotten about this PR, now I will fix it.

Reply:
ping @cavallium

Reply:
Done

Reply:
Sorry for the delay, I had a lot of work to do these weeks